src/app/test_executive/test_executive.ml

```diff
@@ -1,50 +1,62 @@
 open Core
 open Async
 open Cmdliner
+open Integration_test_lib
 
-type inputs = {coda_image: string; coda_automation_location: string}
+type test = string * (module Test_functor)
 
-let network_config_from_inputs {coda_image; _} =
-  let open Network_config.Abstract in
-  let open Network_config.Abstract.Images in
-  let bal = Currency.Balance.of_string in
-  { images=
-      { coda= coda_image
-      ; user_agent= "codaprotocol/coda-user-agent:0.1.5"
-      ; bots= "codaprotocol/coda-bots:0.0.13-beta-1"
-      ; points= "codaprotocol/coda-points-hack:32b.4" }
-  ; cloud= Cloud.default
-  ; k= 20
-  ; delta= 3
-  ; proof_level= Runtime_config.Proof_keys.Level.Full
-  ; txpool_max_size= 3000
-  ; block_producers=
-      [{balance= bal "7500"}; {balance= bal "5000"}; {balance= bal "2500"}]
-  ; num_snark_workers= 4
-  ; snark_worker_fee= "0.025"
-  ; snark_worker_public_key=
-      "4vsRCVQZ41uqXfVVfkBNUuNNS7PgSJGdMDNAyKGDdU1WkdxxyxQ7oMdFcjDRf45fiGKkdYKkLPBrE1KnxmyBuvaTW97A5C8XjNSiJmvo9oHa4AwyVsZ3ACaspgQ3EyxQXk6uujaxzvQhbLDx"
-  }
+type inputs = {test: test; coda_image: string}
```

Comment:
should `Test_functor` be `Test_functor_S` or such to indicate it's a signature?

Reply:
`Test_functor_S` seems weird to me. Perhaps `Test_functor_intf` fits our current pattern better?

Reply:
sure